CHANGELOG.md

```diff
@@ -1,5 +1,16 @@
 # Changelog
 
+## 2.14.0
+
+### Changed
+
+* bumps `application/vnd.elife.article-vor+json` from `version=6` to `version=7` in the API
+    - version 7 article-vor responses may contain elifeAssessment, recommendationsForAuthors and publicReviews
```

Comment:
these are new and go under the `### Added` header

Reply:
typo, it's `recommendationForAuthors` not `recommendationsForAuthors`.

Reply:
The typo is in the schema and sample. Should be `recommendationsForAuthors`. I will update today.